README.md

```diff
@@ -850,26 +850,27 @@ You can call it multiple times for different layers.
 Example: `set_global_routing_layer_pitch Metal6 1.34`.
 
 ```
-set_clock_routing [-clock_pdrev_fanout fanout] \
-                  [-clock_topology_priority priority]
+set_clock_routing [-pdrev_fanout fanout] \
+                  [-pdrev_alpha alpha]
 ```
 The `set_clock_routing` command sets specific configurations for clock nets.
 Options description:
-- **clock_pdrev_fanout**: Set the minimum fanout to use PDRev for the routing topology construction of the clock nets (e.g.: -clock_pdrev_fanout 5)
-- **clock_topology_priority**: Set the PDRev routing topology construction priority for clock nets.
-See `set_pdrev_topology_priority` command description for more details about PDRev and topology priority (e.g.: -topology_priority 0.6)
+- **pdrev_fanout**: Set the minimum fanout to use PDRev for the routing topology construction of the clock nets (e.g.: -pdrev_fanout 5)
+- **pdrev_alpha**: Set the PDRev routing topology construction trade-off for clock nets. See `set_pdrev_alpha` command description for
+more details about PDRev and topology trade-off.
+Example: `set_clock_routing -pdrev_fanout 5 -pdrev_alpha 0.6`
 
 ```
-set_pdrev_topology_priority netName alpha
+set_pdrev_alpha [-net net_name] alpha
 ```
 FastRoute has an alternative tool for the routing topology construction, called PDRev. You can define the topology construction
-priority of PDRev between wire length and skew, using the `alpha` parameter.
-The `set_pdrev_topology_priority` command sets the PDRev routing topology construction priority for specific nets.
-Alpha is a positive float between 0.0 and 1.0, where alpha close to 0.0 generates topologies with shorter wire length,
-and alpha close to 1.0 generates topologies with lower skew. For more information about PDRev, check the paper in
-`src/FastRoute/src/pdrev/papers/PDRev.pdf`
+trade-off between minimum wire length and path length between the driver and the loads, using the `alpha` parameter.
+The `set_pdrev_alpha` command sets the PDRev routing topology construction trade-off for all nets or for a specific with the `-net` option.
```

Comment:
This does not do a good job of saying whether set_pdrev_alpha enables pdrev for all nets.
The command name implies it only sets the parameter used by pdrev.

The 'set_pdrev_alpha' command enables the Prim/Dijkstra algorithm and sets the alpha parameter used to trade-off wire length and path depth.